.gitattributes

```diff
@@ -1,3 +1,5 @@
 # Github language settings
 *.h linguist-language=c
 *.c linguist-language=c
+
+* text=auto
```

Comment:
I'd prefer not to do any conversions on any files in the repo as it gets messy as *what you see (in your file browser and text editors) is **not** what you get* :cry:. Up until now this wasn't needed and I think it should stay so. It should actually work correctly even on your setup without this option (pretty much as it works for other files in `src/` as you wrote).

Reply:
I've amended it locally and will push the changes once the other issues are resolved.